tck/src/main/java/org/eclipse/microprofile/rest/client/tck/ClientHeaderParamTest.java

```diff
@@ -66,9 +67,14 @@ private static void stub(String expectedHeaderName, String... expectedHeaderValu
         String expectedIncomingHeader = Arrays.stream(expectedHeaderValue)
                                               .collect(Collectors.joining(","));
         String outputBody = expectedIncomingHeader.replace(',', '-');
+        MappingBuilder mappingBuilder = get(urlEqualTo("/"));
+
+        // headers can be sent either in a single line with comma-separated values o in multiple lines
```

Comment:
```suggestion
        // headers can be sent either in a single line with comma-separated values or in multiple lines
```

just a minor type "o" should be "or"